src/components/OrgSettings/ActionItemCategories/Modal/CategoryViewModal.tsx

```
@@ -0,0 +1,116 @@
+
+  return (
+    <Modal className={styles.itemModal} onHide={hide} show={isOpen}>
+      <Modal.Header>
+        <p className={styles.titlemodal}>{t('categoryDetails')}</p>
+        <Button
+          variant="danger"
+          onClick={hide}
+          className={styles.closeButton}
+          data-testid="categoryViewModalCloseBtn"
+        >
+          <i className="fa fa-times"></i>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)